static/app/views/settings/components/dataSecrecy/index.tsx

```diff
@@ -140,9 +172,9 @@ export default function DataSecrecy() {
       <PanelBody>
         {!allowAccess && (
           <PanelAlert>
-            {hasValidTempAccess
+            {allowData?.accessEnd && moment().isBefore(moment(allowData.accessEnd))
```

Comment:
`hadValidTempAccess` was a a constant which wasn't stateful, so just moved the stateful conditional here.